x-pack/plugins/security_solution/public/common/components/use_combo_box_reset/index.test.tsx

```diff
@@ -63,23 +63,25 @@ describe('useEuiComboBoxReset', () => {
       );
     };
 
-    render(<EuiComboBoxResetTest />);
+    const { debug } = render(<EuiComboBoxResetTest />);
 
-    const initialValue = screen.getByTestId('comboBoxInput'); // EuiComboBox does NOT render the current selection via it's input; it uses this div
-    expect(initialValue).toHaveTextContent(options[0].label);
+    debug();
```

Comment:
please remove debug!

Reply:
Shoot, thank you so much for catching this Steph!

Reply:
https://github.com/elastic/kibana/pull/170716/commits/c85649b74972cf710465d083d1b926cb87f9610e